golang/cosmos/app/app.go

```diff
@@ -963,10 +978,16 @@ func (app *GaiaApp) initController(ctx sdk.Context, bootstrap bool) {
 	app.controllerInited = true
 
 	// Begin initializing the controller here.
+	var swingsetConfig map[string]any
+	if resolvedConfig, ok := app.resolvedConfig.(*viper.Viper); ok {
+		swingsetSection := resolvedConfig.AllSettings()[swingset.ConfigPrefix]
+		swingsetConfig, _ = swingsetSection.(map[string]any)
+	}
```

Comment:
Why is this not using `SwingsetConfigFromViper` in any way?

I don't see how any resolutions that may be performed would be carried here without it.

Reply:
This is trivial right now, but I'm expecting to replace it with use of a function that will probably be defined in config.go. But it won't be `SwingsetConfigFromViper`, because that function is limited to the `[swingset]` section whereas this future one will be consuming full configuration and emitting the superset.

Reply:
Update: It _is_ now `SwingsetConfigFromViper`.